command/watch.go

```diff
@@ -0,0 +1,131 @@
+package command
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/url"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/gotify/cli/v2/config"
+	"github.com/gotify/cli/v2/utils"
+	"github.com/gotify/go-api-client/v2/models"
+	"gopkg.in/urfave/cli.v1"
+)
+
+func Watch() cli.Command {
+	return cli.Command{
+		Name:      "watch",
+		Usage:     "watch the result of a command and pushes output difference",
+		ArgsUsage: "<cmd>",
+		Flags: []cli.Flag{
+			cli.Float64Flag{Name: "interval,n", Usage: "watch interval (sec)", Value: 2.},
```

Comment:
```suggestion
			cli.Float64Flag{Name: "interval,n", Usage: "watch interval (sec)", Value: 2},
```

Reply:
It's my habit to add a decimal point to floats to make it more obvious that it is a float type ;D